src/features/adr/pages/ADRForm.tsx

```diff
@@ -14,10 +14,13 @@ const useStyles = stylesFactory({
     marginRight: "auto",
     maxWidth: "100%",
   },
-  input: { display: "none" },
+
   paper: {
     padding: 20,
     maxWidth: 600,
+    minHeight: "100vh",
```

Comment:
the `Home.tsx` is still showing some footer space.
This is super clumsy.. but you could change one of the `30px` to `4px` and that fits:
`<Box height="calc(100vh - 4px - 72px - 30px - 30px)">`

Reply:
Oh yeah!
29ec664